### DISCUSSION TITLE
2008/06/23/%E7%BB%88%E4%BA%8E%E8%AF%BB%E5%AE%8C%E4%BA%86%E6%95%B4%E6%9C%AC%E3%80%8A%E5%BF%83%E7%90%86%E5%AD%A6%E3%80%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#225] [Announcements]
# 2008/06/23/%E7%BB%88%E4%BA%8E%E8%AF%BB%E5%AE%8C%E4%BA%86%E6%95%B4%E6%9C%AC%E3%80%8A%E5%BF%83%E7%90%86%E5%AD%A6%E3%80%8B

这是我这几年读的时间最长的一本书（大约读了两年），一是因为它特别厚（699页），再有这本书是断断续续读的。我这本《心理学》的作者是 David G. Myers，是美国比较流行的一部心理学导论教材，我看到的是它的第七版。

https://blog.qizhen.xyz/2008/06/23/%E7%BB%88%E4%BA%8E%E8%AF%BB%E5%AE%8C%E4%BA%86%E6%95%B4%E6%9C%AC%E3%80%8A%E5%BF%83%E7%90%86%E5%AD%A6%E3%80%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Nicholas
2008年07月5日 11:59 编辑
一巴掌拍在马腿上的时候多了，原来是没看过心理学，这个，等回去东方书城买一本来看看？

回复
Qizhen
2008年07月1日 14:32 编辑
是啊，说明LabVIEW读者不多啊。

回复
arrowpig
2008年06月29日 21:01 编辑
阮奇祯你有没有发现，非技术文章的读者总是比技术文章的读者起劲

回复
竹贤
2008年06月27日 08:40 编辑
待人接物必须有几个最基本的前提：待人要真诚、诚信；要与人为善；要有爱心、宽容心、报恩心。如同俗话所说：将心比心。
在这前提下，多学习一点“心理学”等知识当然好啦。

回复
杰
2008年06月24日 18:42 编辑
呵呵。
学习总是好的，改天弄一本看看，
不过关于老婆的行为，我认为不可预测，而且，毫无规律可循。哈

回复
ruirui
2008年06月24日 02:13 编辑
心理学的书我也看，但是目的不是指导未来的行动。
只是用来安慰自己，原来我是受这些生物正常的心理反应影响的而不是自己犯傻，哈哈。

回复
磊
2008年06月23日 23:57 编辑
也去买来看看~

回复
Bin
2008年06月23日 21:51 编辑
我以前读过《心理学与生活》这本书，理查德.格里格写的，是美国ETS推荐为GRE心理学考试的主要参考用书。很不错，也是一眼看到就买的。

回复
永青
2008年06月23日 21:49 编辑
这本书看完对理解老婆的心思是没有太多帮助的。。。

回复
Haiying
2008年06月23日 21:08 编辑
是不是那本教科书啊？

回复
